lib/modLocations.js

```diff
@@ -2,43 +2,42 @@ const {hash, parseChance} = require("./utils.js")
 
 module.exports = function($) {
     const table = $("#modLocations").next("table")
-
     const tbody = table.children()['0']
 
     let mod = null
-
     let mods = []
 
-    for(let tr of tbody.children) {
+    for (let tr of tbody.children) {
         let elem = tr.children[0]
         let text = $(elem).text()
 
-        if(elem.name === "th" && tr.children.length === 1) {
-
-            if(mod) {
+        if (elem.name === "th" && tr.children.length === 1) {
+            if (mod) {
+                mod._id = hash(`${mod.modName} ${mod.enemies.length}`)
                 mods.push(mod)
             }
 
-            mod = {_id: hash(text), modName: text, enemies: []}
+            mod = {modName: text, enemies: []}
         }
 
-        if(elem.name === "td" && elem.attribs.class !== "blank-row") {
+        if (elem.name === "td" && elem.attribs.class !== "blank-row") {
             let chanceElem = tr.children[2]
             let chance = parseChance($(chanceElem).text())
-
             let enemyModDropChance = $(tr.children[1]).text()
+            enemyModDropChance = Number(enemyModDropChance.slice(0, enemyModDropChance.length - 1))
 
             mod.enemies.push({
-                _id: hash(text),
+                _id: hash(`${text} ${enemyModDropChance} ${chance.rarity}`),
```

Comment:
now there's spaces, which shouldn't be there....

Reply:
This doesn't matter to be fair, as the output will be the same length and will change no matter what due to the earlier changes. Let's just wait for the other conversation to finish; whether this is truly a good idea and all.